applets/wncklet/workspace-switcher.c

```diff
@@ -522,7 +672,9 @@ static void display_workspace_names_changed(GSettings* settings, gchar* key, Pag
 
 static void all_workspaces_changed(GSettings* settings, gchar* key, PagerData* pager)
 {
-	gboolean value = g_settings_get_boolean (settings, key);
+	gboolean value = TRUE; /* Default value */
+
+	value = g_settings_get_boolean (settings, key);
```

Comment:
Seems like a rebase mishap

```suggestion
	gboolean value = g_settings_get_boolean (settings, key);
```

Reply:
I see that, not sure how that happened but will fix it. Since` value` it will be unconditionally assigned whatever comes back from `gboolean value = g_settings_get_boolean (settings, key);` even if that somehow turns out to be garabge, the initialization to TRUE should never be used, so we save one assignment here

Reply:
Fixed and force-pushed, thanks for catching it!